packages/prism-theme-odyssey/src/theme.scss

```diff
@@ -4,22 +4,22 @@
 
 :root {
   --font-size: #{$size-body-base};
-  --inlineCodeColor: #{cv('purple', '500')};
+  --inlineCodeColor: #{cv('blue', '000')};
   --inlineCodeBackground: #{cv('red', '300')};
-  --blockBackground: #{cv('gray', '000')};
-  --baseColor: #{cv('gray', '900')};
-  --selectedColor: #{cv('blue', '300')};
-  --highlightBackground:  #{cv('yellow', '000')};
-  --highlightAccent:  #{cv('yellow', '300')};
+  --blockBackground: #{cv('gray', '900')};
+  --baseColor: #{cv('blue', '000')};
+  --selectedColor: #{cv('turquoise', '500')};
+  --highlightBackground: #{cv('yellow', '000')};
+  --highlightAccent: #{cv('yellow', '300')};
   --commentColor: #{cv('gray', '500')};
-  --punctuationColor: #{cv('gray', '200')};
-  --propertyColor: #{cv('purple', '500')};
-  --selectorColor: #{cv('turquoise', '500')};
-  --operatorColor: #{cv('yellow', '900')};
+  --punctuationColor: #{cv('blue', '000')};
+  --propertyColor: #{cv('turquoise', '400')};
+  --selectorColor: #a0dcc3;
+  --operatorColor: #{cv('blue', '000')};
   --operatorBg: transparent;
-  --variableColor: #{cv('green', '900')};
-  --functionColor: #{cv('red', '500')};
-  --keywordColor: #{cv('blue', '500')};
+  --variableColor: #{cv('blue', '400')};
+  --functionColor: #cfb1c1;
```

Comment:
Same as above, maybe we can quarantine them as "Brand Only" until @joejezowski-okta is finished elaborating the available palette.

Reply:
See: https://github.com/okta/odyssey/pull/919#discussion_r604478868